src/employees/components/RegisterEmployeeForm.vue

```diff
@@ -0,0 +1,152 @@
+                outlined
+              ></v-text-field>
+            </v-col>
+            <v-col>
+              <v-text-field
+                v-model="employeeInfo.lastName"
+                :rules="nameRules"
+                label="Last name"
+                required
+                outlined
+              ></v-text-field>
+            </v-col>
+          </v-row>
+        </v-card-text>
+      </v-card>
+
+      <v-card class="mb-10">
+        <v-card-title>Contact info</v-card-title>
+        <v-card-text>
+          <v-row>
+            <v-col>
+              <v-text-field
+                v-model="employeeInfo.email"
+                :rules="emailRules"
+                label="Email"
+                required
+                outlined
+              ></v-text-field>
+            </v-col>
+
+            <v-col>
+              <v-text-field
+                v-model="employeeInfo.phoneNumber"
+                label="Phone number"
+                type="phone"
+                :rules="phoneNumberRules"
+                required
+                outlined
+              ></v-text-field>
+            </v-col>
+          </v-row>
+
+          <v-row>
+            <v-col>
+              <v-text-field
+                v-model="employeeInfo.address"
+                label="Address"
+                required
+                outlined
+              ></v-text-field>
+            </v-col>
+
+            <v-col>
+              <v-menu
+                v-model="calendarOpen"
+                :close-on-content-click="false"
+                transition="scale-transition"
+                offset-y
+                max-width="290px"
+                min-width="auto"
+              >
+                <template v-slot:activator="{ on, attrs }">
+                  <v-text-field
+                    v-model="employeeInfo.birthDate"
+                    label="Birth date"
+                    outlined
+                    persistent-hint
+                    append-icon="mdi-calendar"
+                    readonly
+                    v-bind="attrs"
+                    v-on="on"
+                  ></v-text-field>
+                </template>
+                <v-date-picker
+                  v-model="employeeInfo.birthDate"
+                  no-title
+                  @input="calendarOpen = false"
+                ></v-date-picker>
+              </v-menu>
+            </v-col>
+          </v-row>
+        </v-card-text>
+      </v-card>
+
+      <v-btn color="#72418b" dark @click="onSubmit()" x-large>CREATE</v-btn>
+    </v-form>
+  </div>
+</template>
+
+<script>
+import validators from "@/shared/form-validators";
+export default {
+  data() {
+    return {
+      calendarOpen: false,
+      employeeInfo: {
+        firstName: "",
+        lastName: "",
+        email: "",
+        phoneNumber: "",
+        address: "",
+        birthDate: "",
+      },
+      formValidity: false,
+      nameRules: [validators.required("Name company is required")],
```

Comment:
`Employee name` NOT `Name company`

but there are first name and last name, therefore should have two rules, one for the first name and one for the last name

```suggestion
      firstNameRules: [validators.required("First name is required")],
      lastNameRules: [validators.required("Last name is required")],
```